cider/type/plan/Analyzer.h

```diff
@@ -323,6 +334,9 @@ class ColumnVar : public Expr {
   bool operator==(const Expr& rhs) const override;
   std::string toString() const override;
 
+ public:
+  JITExprValue& codegen(JITFunction& func) override;
```

Comment:
Should we move these ```Expr``` subclasses to separate files step-by-step? Maybe ```Analyzer.h``` is oversized.

Reply:
good idea